### DISCUSSION TITLE
Json message is truncated above 16KB size

### DISCUSSION THREAD

**Discussion by alwalaupendra** [ashie/td-agent-builder#769] [Q&A]
Hi Team,
We are using fluentd is a shipping agent for sending logs from application team to logstash server. We are not getting total json message which is above 16kb in size. I am suspecting it is truncated from fluentd side. Could you please guide me how can we get total json message without truncate?

**Comment by daipom** [comment]
Sorry for our late response.
Could you share the configuration?
It is possible that one of the plugins you are using is truncating the data.
